src/components/FileUploader/FileUploader.js

```diff
@@ -31,14 +28,13 @@ const FileUploader = ({
         setOpen(false);
         setFilename(files[0].name);
         onSave(files);
+        console.log(filename);
```

Comment:
Let's not try to have console logs in the code once you are done debugging locally.